airflow/providers/amazon/aws/hooks/glue_crawler.py

```diff
@@ -15,13 +15,15 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+"""This module contains AWS Glue Crawler Hook"""
```

Comment:
This docstring is useless. I’d suggest either removing it or provide some better explaination.